recipes/faampy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "faampy" %}
+{% set version = "v0.1.4" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: faampy-{{ version }}.tar.gz
+    url: https://github.com/ncasuk/faampy/archive/{{ version }}.tar.gz
+    sha256: 9540646948656051d484e17b0df2fc02c83eff411137f956a7a2f523e0eb30dc
+
+build:
+    skip: True  # [py3k]
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+
+    run:
+        - python
+        - basemap
+        - cartopy
+        - configparser
+        - ephem
+        - matplotlib >=1.4.0
+        - setuptools
+        - netcdf4 >=1.0
+        - numpy
+        - scipy
+        - pandas
+        - gdal
+        - exifread
+        - fiona
+        - gdal
```

Comment:
I recommend you to pin `gdal`  to `2.2.*` to avoid headaches. Or better yet, remove `gdal` from the dependencies and let `fiona` take care of it.